pkg/syncer/storagepool/storageclasswatch.go

```diff
@@ -250,56 +255,57 @@ func (w *StorageClassWatch) refreshStorageClassCache(ctx context.Context) error
 
 // We want each StorageClass to have an annotation capturing the content of the
 // SPBM storage policy. This function makes it so.
-func (w *StorageClassWatch) addStorageClassPolicyAnnotation(ctx context.Context,
-	profile cnsvsphere.SpbmPolicyContent) error {
+// TODO: we are not performing any updates because the feature switch is always off. This code needs to be refactored.
```

Comment:
if feature switch is always off, can we remove this dead code? why do we need to make change in this method when it is not used due to switch is off.

Reply:
Done.